src/index.js

```diff
@@ -45,11 +43,22 @@ class MCServer extends EventEmitter {
   }
 
   connect (options) {
-    const version = require('minecraft-data')(options.version).version
-    if (!supportedVersions.some(v => v.includes(version.majorVersion))) {
-      throw new Error(`Version ${version.minecraftVersion} is not supported.`)
+    const mcData = require('minecraft-data')(options.version)
+    const version = mcData.version
+    const { 0: maxMajor, 1: maxMinor, 2: maxPatch = 0 } = latestSupportedVersion.split('.')
+    const { 0: minMajor, 1: minMinor, 2: minPatch = 0 } = oldestSupportedVersion.split('.')
+    const { 0: major, 1: minor, 2: patch = 0 } = version.minecraftVersion.split('.')
```

Comment:
this will not work for snapshot version. Why not use the same logic as in mineflayer?

Reply:
You linked to the wrong file; I wasn't able to easily find it so I wrote my own.

I'll go look again, but harder.

Reply:
Never mind. The naming confused me. I'll implement it now.